lib/mongoose_push/service/apns/supervisor.ex

```diff
@@ -32,6 +32,12 @@ defmodule MongoosePush.Service.APNS.Supervisor do
           {MongoosePush.Service.APNS.State, pool_configs}
         ]
 
+        :telemetry.execute(
+          [:mongoose_push, :apns_supervisor, :init],
```

Comment:
`init` events should always get their `terminate` counterpart in order to enable metrics like "number of active instances".

Reply:
Here we use `Supervisor` and not `GenServer` and it doesn't have a `terminate` callback - not sure how to handle that